localstack/deprecations.py

```diff
@@ -53,11 +53,6 @@ def is_affected(self) -> bool:
         "0.12.7",
         "LocalStack now has full multi-region support. Please remove this environment variable.",
     ),
-    EnvVarDeprecation(
-        "DEFAULT_REGION",
-        "0.12.7",
-        "LocalStack now has full multi-region support. Please remove this environment variable.",
-    ),
```

Comment:
I think this deprecation should be kept, but rephrased such that it says that it does not have any effect anymore.